apps/server/src/app/middleware/validate-auth-jwt.ts

```diff
@@ -2,17 +2,20 @@ import cookieParser from 'cookie-parser'
 import { decode } from 'next-auth/jwt'
 
 const SECRET = process.env.NEXTAUTH_SECRET ?? 'REPLACE_THIS'
-
 export const validateAuthJwt = async (req, res, next) => {
     cookieParser(SECRET)(req, res, async (err) => {
         if (err) {
             return res.status(500).json({ message: 'Internal Server Error' })
         }
 
-        if (req.cookies && 'next-auth.session-token' in req.cookies) {
+        const cookieName = req.secure
```

Comment:
@6ixfalls I think this will address the issue we were talking about

Reply:
Alternatively you could also just force the cookie to be secure on the next auth side, which is what I did but this is fine.